Mathlib/Order/Atoms.lean

```diff
@@ -810,20 +836,12 @@ end IsSimpleOrder
 
 namespace IsSimpleOrder
 
-variable [CompleteLattice α] [IsSimpleOrder α]
-
---set_option default_priority 100
--- Porting note: not supported, done for each instance individually
+variable [PartialOrder α] [BoundedOrder α] [IsSimpleOrder α]
 
-instance (priority := 100) : IsAtomistic α :=
-  ⟨fun b =>
-    (eq_bot_or_eq_top b).elim
-      (fun h => ⟨∅, ⟨h.trans sSup_empty.symm, fun _ ha => False.elim (Set.not_mem_empty _ ha)⟩⟩)
-      fun h =>
-      ⟨{⊤}, h.trans sSup_singleton.symm, fun _ ha =>
-        (Set.mem_singleton_iff.1 ha).symm ▸ isAtom_top⟩⟩
+instance (priority := 100) : IsAtomistic α where
+  isLUB_atoms b := (eq_bot_or_eq_top b).elim (fun h ↦ ⟨∅, by simp [h]⟩) (fun h ↦ ⟨{⊤}, by simp [h]⟩)
 
-instance : IsCoatomistic α :=
+instance (priority := 100) : IsCoatomistic α :=
```

Comment:
For reviewers: this priority is new, but the mathlib3 version had this meaning, and the comment previously on line 816 indicates that this was the intended priority. This also means the priority for the atomistic and coatomistic instances are matching, rather than the coatomistic version arbitrarily having higher priority.